nmmo/systems/item.py

```diff
@@ -371,6 +380,13 @@ def damage(self):
 # NOTE: Each consumable item (ration, poultice) cannot be stacked,
 #   so each item takes 1 inventory space
 class Consumable(Item):
+
+  def __init__(self, realm, level, **kwargs):
+    # ignore quantity for Consumable, so that the default quantity of 1 is used
+    if 'quantity' in kwargs:
```

Comment:
I'm concerned that we have to do this in a bunch of places, could you explain the problem? Maybe there's a cleaner solution?

Reply:
Thinking further, I think a cleaner way would be removing quantity arg and setting quantity=1 by default. There might be cases where we may want to init Ammo with quantity > 1, and it's only Ammo. So perhaps, I should add quantity only to Ammo and see if this could work.

Reply:
That sounds good, could you make that change? The way you have it is too hacky

Reply:
Will do after checking with Joseph on `droptable.py`